tex/chapters/electric-field/introduction.tex

```diff
@@ -0,0 +1,14 @@
+\section{Introduction}
+Let $q$ and $Q$ be the charges of two point-like particles, located at positions $\mathbf{r_{q}}$ and $\mathbf{r_{Q}}$, respectively.
+According to Coulomb's law, the force experienced by $q$ due to $Q$ is:
+\be\label{eq:culombs-law}
+\mathbf{F}_{q,Q}(\mathbf{r}_q)=\frac{qQ}{4\pi\epsilon_0}\frac{\mathbf{r}_{q}-\mathbf{r}_{Q}}{\norm{\mathbf{r}_{q}-\mathbf{r}_{Q}}^{3/2}}
+\ee
+It is clear that the quantity $\mathbf{F}_{q,Q}/q$ does not depend on $q$. This is true under the assumption that the presence of $q$ does not influence the mathematical model provided in \ref{eq:culombs-law}, in particular, if $q$ is infinitely small. Under this assumption, a formal definition of field is provided in by the following.
+\be\label{eq:electric-field}
+\mathbf{E}(\mathbf{r}_{q})=\lim_{q \to 0} \frac{\mathbf{F}_{Q,q}}{q}
+\ee
+At any point $\mathbf{r}_{q}$, \ref{eq:electric-field} is the electric field generated by a point-like particle with charge $Q$, located at $\mathbf{R}$. From \ref{eq:culombs-law}, it is clear that the relative location $\mathbf{r}_{q}-\mathbf{r}_{Q}$ governs the filed calculation. It is therefore convenient to define the vector $\mathbf{r}=\mathbf{r}_{q}-\mathbf{r}_{Q}$ and write \ref{eq:electric-field} as 
```

Comment:
change `$\mathbf{R}$` to `$\mathbf{r}_{Q}$

Reply:
Done.